<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1096</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QCustomPlot plot examples</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCustomPlot" name="customPlot" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1096</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="action_Close"/>
    <addaction name="separator"/>
    <addaction name="actionLoad_Mask"/>
   </widget>
   <widget class="QMenu" name="menu_Graph">
    <property name="title">
     <string>&amp;Graph</string>
    </property>
    <addaction name="action_reset"/>
    <addaction name="action_screenshot"/>
   </widget>
   <widget class="QMenu" name="menuImageSDK">
    <property name="title">
     <string>ImageSDK</string>
    </property>
    <addaction name="action_Qt"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuImageSDK"/>
   <addaction name="menu_Graph"/>
  </widget>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="action_reset">
   <property name="text">
    <string>&amp;reset</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="action_screenshot">
   <property name="text">
    <string>&amp;screenshot</string>
   </property>
  </action>
  <action name="action_Qt">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Qt</string>
   </property>
  </action>
  <action name="actionLoad_Mask">
   <property name="text">
    <string>Load&amp;Mask</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>action_Close</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>547</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
